test/stream_test.py

```diff
@@ -15,8 +15,7 @@
 
 class StreamTest(unittest.TestCase):
     zk_host = "127.0.0.1:2181"
-    broker_port = 1101
-    broker_address = "127.0.0.1"
+    broker_host = "127.0.0.1:1101"
```

Comment:
ditto.

Reply:
That line is nothing to do with the above comment. It simply represents the broker host as a string of the {ip:port} format.